supabase/seed/clientsSeed.mts

```diff
@@ -37,3 +37,5 @@ export const possibleParcelPostCodes = [
     "CB8 9LJ",
     null,
 ];
+
+export const booleansWeightedToTrue = [true, true, true, true, true, true, true, false];
```

Comment:
Maybe like this? `Array(8).fill(true).concat(Array(1).fill(false))` 
Just makes the ratio a bit more obvious imo

Reply:
that's good, thank you!